packages/core/src/render3/errors.ts

```diff
@@ -70,6 +75,21 @@ export function throwErrorIfNoChangesMode(
   throw new RuntimeError(RuntimeErrorCode.EXPRESSION_CHANGED_AFTER_CHECKED, msg);
 }
 
+function formatValue(value: any): string {
```

Comment:
We can probably use `unknown` here:
```suggestion
function formatValue(value: unknown): string {
```

Reply:
Of course !